setup.py

```diff
@@ -80,71 +91,71 @@
         'console_scripts': [
             'mintpy = mintpy.__main__:main',
             'add_attribute.py = mintpy.add_attribute:main',
-            'add.py = mintpy.add:main',
-            'asc_desc2horz_vert.py = mintpy.asc_desc2horz_vert:main',
-            'bulk_plate_motion.py = mintpy.bulk_plate_motion:main',
-            'closure_phase_bias.py = mintpy.closure_phase_bias:main',
-            'dem_error.py = mintpy.dem_error:main',
-            'dem_gsi.py = mintpy.dem_gsi:main',
-            'diff.py = mintpy.diff:main',
-            'generate_mask.py = mintpy.generate_mask:main',
-            'geocode.py = mintpy.geocode:main',
-            'ifgram_inversion.py = mintpy.ifgram_inversion:main',
-            'image_math.py = mintpy.image_math:main',
-            'image_stitch.py = mintpy.image_stitch:main',
-            'info.py = mintpy.info:main',
-            'iono_tec.py = mintpy.iono_tec:main',
-            'load_data.py = mintpy.load_data:main',
-            'load_gbis.py = mintpy.load_gbis:main',
-            'local_oscilator_drift.py = mintpy.local_oscilator_drift:main',
-            'lookup_geo2radar.py = mintpy.lookup_geo2radar:main',
-            'mask.py = mintpy.mask:main',
-            'modify_network.py = mintpy.modify_network:main',
-            'multilook.py = mintpy.multilook:main',
+            'add.py = mintpy.cli.add:main',
+            'asc_desc2horz_vert.py = mintpy.cli.asc_desc2horz_vert:main',
+            'bulk_plate_motion.py = mintpy.cli.bulk_plate_motion:main',
+            'closure_phase_bias.py = mintpy.cli.closure_phase_bias:main',
+            'dem_error.py = mintpy.cli.dem_error:main',
+            'dem_gsi.py = mintpy.cli.dem_gsi:main',
+            'diff.py = mintpy.cli.diff:main',
+            'generate_mask.py = mintpy.cli.generate_mask:main',
+            'geocode.py = mintpy.cli.geocode:main',
+            'ifgram_inversion.py = mintpy.cli.ifgram_inversion:main',
+            'image_math.py = mintpy.cli.image_math:main',
+            'image_stitch.py = mintpy.cli.image_stitch:main',
+            'info.py = mintpy.cli.info:main',
+            'iono_tec.py = mintpy.cli.iono_tec:main',
+            'load_data.py = mintpy.cli.load_data:main',
+            'load_gbis.py = mintpy.cli.load_gbis:main',
+            'local_oscilator_drift.py = mintpy.cli.local_oscilator_drift:main',
+            'lookup_geo2radar.py = mintpy.cli.lookup_geo2radar:main',
+            'mask.py = mintpy.cli.mask:main',
+            'modify_network.py = mintpy.cli.modify_network:main',
+            'multilook.py = mintpy.cli.multilook:main',
             'multi_transect.py = mintpy.multi_transect:main',
```

Comment:
Why's this one the only untouched one?

Reply:
Multi-transect uses the old getopt module instead of argparse so it is not possible to integrate it in the current "single-entrypoint" without a porting.
I could try to doit but I was not sure if `multi_transect` is actually used and updated or something legacy.
Please let me know.

Reply:
`multi_transect` is a legacy script and has not been actively maintained. Keeping it out of this PR is perfectly fine with me.